nixos/modules/security/acme/default.nix

```diff
@@ -196,30 +195,30 @@ let
     # FIXME when mkChangedOptionModule supports submodules, change to that.
     # This is a workaround
     extraDomains = data.extraDomainNames ++ (
-      optionals
-      (data.extraDomains != "_mkMergedOptionModule")
+      lib.optionals
+      (data.extraDomains != "_lib.mkMergedOptionModule")
```

Comment:
I very much doubt this is correct. If the test passes then we should probably fix it to exercise this code path.

Reply:
I had a pending comment for the same 👍